### PR TITLE
Added/improved a couple of comments to better explain what is going on.

### PR DESCRIPTION
**Pull request by stylesuxx**

### CODE REVIEW

**Review by damosvil — APPROVED** [review]

**Comment by stylesuxx** [comment]
Hey @damosvil - it's just a draft for now. I want to go through the rest of the files too and add comments, just to makes sure I can follow everything too ;-)

File: src/Modules/Commutation.asm [hunk 1]
```diff
@@ -25,14 +25,37 @@
...
-; Performs commutation switching
+; Step   AA' BB' CC'
+; 1 C->A 01  00  10
+; 2 B->A 01  10  00
+; 3 B->C 00  10  01
+; 4 A->C 10  00  01
+; 5 A->B 10  01  00
+; 6 C->B 00  01  10
 ;
+; NOTE: Every steap has a "reverse" step which is used in bi-directional mode
```
Comment (damosvil) [inline src/Modules/Commutation.asm:55]: small typo: steap

**Review by damosvil — APPROVED** [review]

File: src/Modules/Commutation.asm (hunk 1 above)
Comment (stylesuxx) [reply src/Modules/Commutation.asm]: Fixed.

File: src/Bluejay.asm [hunk 2]
```diff
@@ -508,9 +530,13 @@ ELSE
...
 
 bootloader_done:
-    jnb  Flag_Had_Signal, setup_dshot   ; Check if DShot signal was lost
+    ; If we had a signal before, reset the flag, beep, wait a bit and contiune
+    ; with DSHOT setup. If we did not have a signal yet, continue with DSHOT
+    ; setup straight away.
+    jnb  Flag_Had_Signal, setup_dshot
     call beep_signal_lost
-    call wait250ms                      ; Wait for flight controller to get ready
+    ; Wait for flight controller to get ready
```
[the remark this answers is not in the document]
Comment (stylesuxx) [reply src/Bluejay.asm:538]: Fixed.

File: src/Bluejay.asm [hunk 3]
```diff
@@ -611,20 +638,22 @@ arming_begin:
...
 arming_wait:
     clr  C
     mov  A, Rcp_Stop_Cnt
     subb A, #10
-    jc   arming_wait                    ; Wait until rcp has been zero for ~300ms
+    jc   arming_wait
 
     clr  IE_EA
-    call beep_f2_short                  ; Beep signal that ESC is armed
+    call beep_f2_short                  ; Confrim arm state by beeping
```
[the remark this answers is not in the document]
Comment (stylesuxx) [reply src/Bluejay.asm:652]: Fixed.

File: src/Modules/DShot.asm [hunk 4]
```diff
@@ -52,7 +53,23 @@ detect_rcp_level_read:
...
+;
+;  7: Set motor direction to normal
+;  8: Set motor direction to reverse
+;  9: Disable 3D mode
+; 10: Enable 3D mode
+; 12: Save settings
+; 13: Enable EDT (Extended DShot Telemetry)
+; 14: Disable EDT (Extended DShot Telemetry)
+; 20: Set motor direction to user programmed direction
+; 21: Sete motor direction to reversed user programmed direction
```
[the remark this answers is not in the document]
Comment (stylesuxx) [reply src/Modules/DShot.asm:72]: Fixed.

File: src/Modules/Eeprom.asm [hunk 5]
```diff
@@ -154,12 +168,14 @@ read_eeprom_byte:
...
-; Assumes data in address given by Temp1, or in accumulator
-; Assumes address in DPTR
+; ASSERT:
+; - Data in address given by Temp1
+; - Destination address in DPTR
 ;
 ;**** **** **** **** **** **** **** **** **** **** **** **** ****
 write_eeprom_byte:
     mov  A, @Temp1
+; Shorthand if data is already in accumulator
```
[the remark this answers is not in the document]
Comment (stylesuxx) [reply src/Modules/Eeprom.asm:178]: Fixed.

File: src/Modules/Fx.asm [hunk 6]
```diff
@@ -249,10 +251,14 @@ beacon_beep_exit:
...
-; item2 - is the duration of each pulse of the musical note, lower the value, higher the pitch
-; item1 - if item2 is zero, it is the number of milliseconds of wait time, else it is the number of pulses of item2
+; item2 - is the duration of each pulse of the musical note.
+;         The lower the value, the higher the pitch.
+; item1 - if item2 is zero, it is the number of milliseconds of wait time, else
+;         it is the number of pulses of item2.
 ;
-; Requirements: Interrupts must be disabled and FETs turned off
+; Requirements:
+; - Interrupts must be disabledz
```
[the remark this answers is not in the document]
Comment (stylesuxx) [reply src/Modules/Fx.asm:260]: Fixed.

**Pull request merged**